travis/test_docker_image.sh

```diff
@@ -1,29 +0,0 @@
-#!/bin/bash
-
-#the below settings are based on:
-#http://redsymbol.net/articles/unofficial-bash-strict-mode/
-set -euo pipefail
-IFS=$'\n\t'
-
-NETWORK=amoc-arsenal-test-network
-PATH_TO_EXEC=/home/amoc/amoc_arsenal_xmpp/bin/amoc_arsenal_xmpp
-AMOC_NODES="['amoc_arsenal_xmpp@amoc-arsenal-1','amoc_arsenal_xmpp@amoc-arsenal-1']"
-docker network create ${NETWORK}
-
-docker run -t -d --name amoc-arsenal-1 -h amoc-arsenal-1 --network ${NETWORK} \
-           -e AMOC_NODES="${AMOC_NODES}" \
-	       --health-cmd="${PATH_TO_EXEC} status" \
-	       amoc-arsenal-xmpp:latest
-
-docker run -t -d --name amoc-arsenal-2 -h amoc-arsenal-2 --network ${NETWORK} \
-           -e AMOC_NODES="${AMOC_NODES}" \
-	       --health-cmd="${PATH_TO_EXEC} status" \
-	       amoc-arsenal-xmpp:latest
-
-./travis/wait_for_healthcheck.sh amoc-arsenal-1
-./travis/wait_for_healthcheck.sh amoc-arsenal-2
-
-docker exec -it amoc-arsenal-1 ${PATH_TO_EXEC} eval "nodes()" | grep amoc-arsenal-2
-docker exec -it amoc-arsenal-2 ${PATH_TO_EXEC} eval "nodes()" | grep amoc-arsenal-1
-
-
```

Comment:
very minor: extra newline

Reply:
that's an old file, the [new one](https://github.com/esl/amoc-arsenal-xmpp/pull/21/files/a70237e39b1a75ac91a81790302dc5ab8765891b#diff-2fe9639dd0beca7ac77ccd5f3c6806dc4f40faba30c6764e23db74193291ddca) doesn't have that extra new line.